android/tangram/build.gradle

```diff
@@ -46,7 +46,7 @@ afterEvaluate {
 }
 
 dependencies {
-  compile 'com.squareup.okhttp:okhttp:2.5.0'
+  compile 'com.squareup.okhttp3:okhttp:3.1.2'
```

Comment:
As off 25th Feb, there is a newer version **3.2.0** available for okhttp3 on maven. Probably we should move to that.